chapters/05-funcoes-de-ordem-superior.md

```diff
@@ -30,13 +30,13 @@ Qual é mais propenso a erros?
 
 Se juntarmos as definições de `sum` e `range` o tamanho do segundo código será bem maior do que o primeiro, mas eu diria que ele é o mais correto.
 
-A razão dele ser o mais correto é que primeiramente ele constrói uma linguagem que expressa o problema e resolve em seu próprio domínio com uma solução clara. Somar uma sequência de números não se trata de um laço com contadores mas sim de somas e sequências de números.
+A razão dele ser o mais correto é que primeiramente ele constrói uma linguagem que expressa o problema e resolve em seu próprio domínio com uma solução clara. Somar uma sequência de números não se trata de um laço com contadores. Trata-se de intervalos e somas.
 
 A definição do vocabulário (funções `sum` e `range`) terão que lidar com laços, contadores e outros detalhes não muito interessante. Mas por causa deles expressarem conceitos mais simples fica mais fácil de entender.
```

Comment:
**Original:**
*"The definitions of this vocabulary (the functions sum and range) will still involve loops, counters, and other incidental details. But because they are expressing simpler concepts than the program as a whole, they are easier to get right."*

**Tradução:**
As definições desse vocabulário (as funções `sum` e `range`) ainda terão que lidar com laços de repetição, contadores e outros detalhes secundários. No entanto, devido ao fato de representarem conceitos mais simples, elas acabam sendo mais fáceis de se entender.